nncf/experimental/onnx/graph/nncf_graph_builder.py

```diff
@@ -186,10 +187,16 @@ def create_nncf_graph(onnx_model: ModelProto) -> NNCFGraph:
         for node in filter(GraphConverter._is_valid_onnx_metatype, onnx_graph.get_all_nodes()):
             metatype = ONNX_OPERATION_METATYPES.get_operator_metatype_by_op_name(node.op_type)
             layer_attributes = ONNXExtendedLayerAttributes(node.input, node.output)
+            is_shared, layer_name = None, None
+            if metatype in WEIGHT_LAYER_METATYPES:
+                is_shared = onnx_graph.is_node_shared(node)
```

Comment:
Does this mean that we can use `is_shared` instead of `weight_initializer_names = set()` in MinMax algorithm?

Reply:
What's the purpose of checking that node is shared? I mean if I know that node is shared I also need to know whether the corresponding quantizer was added.
Or you have a suggestion?

Reply:
I suggest to separate `non_unified_scales_quantization_points` and ` unified_scales_quantization_points` like in TF part
https://github.com/openvinotoolkit/nncf/blob/develop/nncf/tensorflow/quantization/algorithm.py#L351-L385. 

We should probably do it in another PR.